fml/platform/android/message_loop_android.cc

```diff
@@ -93,4 +108,33 @@ void MessageLoopAndroid::OnEventFired() {
   }
 }
 
+bool MessageLoopAndroid::Register(JNIEnv* env) {
```

Comment:
is it possible to add a comment to indicate that this could just use `AChoreographer` once Flutter drops supports for API level < 24?  

There's not concrete plan, but worth mentioning for future maintainers that using the NDK simplifies this quite a bit.